packages/block-editor/src/store/selectors.js

```diff
@@ -27,12 +27,22 @@ import {
 	hasChildBlocksWithInserterSupport,
 } from '@wordpress/blocks';
 
-/***
- * Module constants
+// Module constants
+/**
+ * @private
  */
 export const INSERTER_UTILITY_HIGH = 3;
+/**
```

Comment:
maybe same spacing here? otherwise good to :ship:

Reply:
4cdecadb11fdc220706d99eca7dbe5322b351047